packages/create-docusaurus/package.json

```diff
@@ -1,21 +1,22 @@
 {
-  "name": "@docusaurus/init",
-  "version": "2.0.0-beta.6",
+  "name": "create-docusaurus",
+  "version": "0.0.6",
```

Comment:
Curious, why "0.0.6"?

Reply:
Thanks

Just because I did some test publish attempt to make sure it works and wanted to keep the ability to increment if needed

This will be fixed on next release by lerna so it doesn't matter